src/lib/helpers/utils.ts

```diff
@@ -54,7 +54,7 @@ export function stringifyUnknown(unknown: unknown): string {
     }
 
     if (unknown instanceof Error) {
-      return jsonStringify(unknown, Object.getOwnPropertyNames(unknown));
+      return `${unknown.message}\n${unknown.stack ?? ""}`;
     }
```

Comment:
What is the reason for this change?
For example Error object could have more properties than just `message` and `stacktrace`

Reply:
The reason is that when the error object is stringified like that it's rendered completely unreadable

I decided improve that according to your recommendation so other properties can be still visible, take a look